gems/aws-sdk-core/lib/aws-sdk-core/errors.rb

```diff
@@ -102,6 +102,10 @@ def initialize(*args)
       end
     end
 
+    # Raised when a credentials provider process returns a JSON
```

Comment:
surprise! ;)  you need to update the region line here: https://github.com/aws/aws-sdk-ruby/blob/master/build_tools/spec/region_spec.rb#L12

That spec is for testing we don't hard code region names, since the error file has been modified, the line number need to be modified as well :D That' why you are seeing the travis error